### PR TITLE
WIP: Add vdatum to image

### PR DESCRIPTION
**Pull request by dpa9694**
## Workflow

* [ ] Closes issue #
* [ ] Notebook:worker pairing builds & passes local build, visual inspection & client.map tests
* [ ] Passes travis tests
* [ ] Image deployed on `dev` branch (see [notebook](https://hub.docker.com/r/rhodium/notebook/tags/) and [worker](https://hub.docker.com/r/rhodium/worker/tags/) dev tags)
* [ ] Worker passes manual deployment test on compute.rhg
* [ ] Notebook+worker passes test-hub deployment
* [ ] Updates integrated into downstream images

## Summary

Provide an overview of your PR here

### Features

* components should be
* enumerated in bullets

### CODE REVIEW

File: base_environment.yml [hunk 1]
```diff
@@ -102,9 +104,9 @@ dependencies:
     - climate-toolbox==0.1.5
     - impactlab-tools==0.4.0
     - parameterize-jobs==0.1.1
-    - git+https://github.com/rhodiumgroup/rhg_compute_tools.git@allow-remote-scheduler#egg=rhg_compute_tools
-    - git+https://github.com/NCAR/intake-esm.git#egg=intake_esm
+    - rhg_compute_tools==0.2.1
+    - git+https://github.com/NCAR/intake-esm.git@v2020.3.16.2#egg=intake_esm
 # need to install from master until 0.10.1
```
Comment (bolliger32) [inline base_environment.yml:109]: let's delete these comments

File: base_environment.yml [hunk 2]
```diff
@@ -102,9 +104,9 @@ dependencies:
...
 # due to handling of remote scheduler
 # (we also should at some point switch to dask-gateway instead of dask-kubernetes)
-    - git+https://github.com/dask/dask-kubernetes.git#egg=dask_kubernetes
+    - dask_kubernetes==0.10.1
```
Comment (bolliger32) [inline base_environment.yml:112]: lets move this up to a conda install (rather than pip)

File: base_environment.yml [hunk 3]
```diff
@@ -102,9 +104,9 @@ dependencies:
     - climate-toolbox==0.1.5
     - impactlab-tools==0.4.0
     - parameterize-jobs==0.1.1
-    - git+https://github.com/rhodiumgroup/rhg_compute_tools.git@allow-remote-scheduler#egg=rhg_compute_tools
-    - git+https://github.com/NCAR/intake-esm.git#egg=intake_esm
+    - rhg_compute_tools==0.2.1
+    - git+https://github.com/NCAR/intake-esm.git@v2020.3.16.2#egg=intake_esm
```
Comment (bolliger32) [inline base_environment.yml:108]: lets install intake-esm from conda now

File: base_environment.yml [hunk 4]
```diff
@@ -2,109 +2,108 @@ name: base
...
 # won't work
-  - esmpy=8.0.0=mpi_mpich_py37ha9b28fa_101 # pinkeep: esmpy=8.0.0=mpi_mpich_py37ha9b28fa_101
-  - fastparquet=0.3.2=py37hc1659b7_0
-  - fiona=1.8.13=py37h900e953_0
-  - fusepy=3.0.1=py_0
+  - esmpy
+  - fastparquet
+  - fiona
+  - fusepy
 # this gcc pin is necessary b/c of a weird feature in the h553295d_15 build
```
Comment (bolliger32) [inline base_environment.yml:29]: if this builds, let's drop this comment

File: base_environment.yml [hunk 5]
```diff
@@ -2,109 +2,108 @@ name: base
...
+  - descartes
+  - distributed
+  - dropbox
 # need to make sure we get esmpy compiled with mpi otherwise xesmf regridding
 # won't work
-  - esmpy=8.0.0=mpi_mpich_py37ha9b28fa_101 # pinkeep: esmpy=8.0.0=mpi_mpich_py37ha9b28fa_101
-  - fastparquet=0.3.2=py37hc1659b7_0
-  - fiona=1.8.13=py37h900e953_0
-  - fusepy=3.0.1=py_0
+  - esmpy
```
Comment (bolliger32) [inline base_environment.yml:25]: if we end up still needing to pin this i think we can do something like esmpy=*=mpi*

File: base_environment.yml [hunk 6]
```diff
@@ -2,109 +2,108 @@ name: base
...
-    - climate-toolbox==0.1.5
-    - impactlab-tools==0.4.0
-    - parameterize-jobs==0.1.1
-    - git+https://github.com/rhodiumgroup/rhg_compute_tools.git@allow-remote-scheduler#egg=rhg_compute_tools
-    - git+https://github.com/NCAR/intake-esm.git#egg=intake_esm
-# need to install from master until 0.10.1
-# due to handling of remote scheduler
-# (we also should at some point switch to dask-gateway instead of dask-kubernetes)
-    - git+https://github.com/dask/dask-kubernetes.git#egg=dask_kubernetes
+    - mapbox
```
Comment (bolliger32) [inline base_environment.yml:103]: drop

File: base_environment.yml [hunk 7]
```diff
@@ -2,109 +2,108 @@ name: base
...
+    - py-noaa
+    - sidecar
+    - climate-toolbox
+    - impactlab-tools
```
Comment (bolliger32) [inline base_environment.yml:107]: drop

File: base_environment.yml [hunk 8]
```diff
@@ -1,110 +1,98 @@
...
+- google-cloud-storage=1.28
+- holoviews=1.13
+- h5netcdf=0.8
+- icu=58.2
+- intake-esm=2020.5
+- iris=2.4
+- jedi=0.17
+- jupyter-server-proxy=1.5
+- kubernetes=1.18
+- lapack=3.6
```
Comment (bolliger32) [inline base_environment.yml:46]: replace this with `blas` which is a "metapackage" in conda and will harmonize the various libraries (like lapack) that rely on blas. we should probably pin it to a specific implementation. So you might want to (either manually or in your environment.yml update utility) pin it to whatever build (e.g. openblas vs mkl) gets selected

File: base_environment.yml [hunk 9]
```diff
@@ -1,110 +1,98 @@
...
+- lz4=3.0
+- make=4.3
+- matplotlib=3.1
+- nc-time-axis=1.2
+- ncurses=6.1
```
Comment (bolliger32) [inline base_environment.yml:51]: drop

File: base_environment.yml [hunk 10]
```diff
@@ -1,110 +1,98 @@
...
+- netcdf-fortran=4.4
+- netcdf4=1.5
+- numba=0.48
+- numcodecs=0.6
```
Comment (bolliger32) [inline base_environment.yml:55]: drop (zarr will take care of this dependency)

File: base_environment.yml [hunk 11]
```diff
@@ -1,110 +1,98 @@
...
+- pandas=1.0
+- phantomjs=2.1
```
Comment (bolliger32) [inline base_environment.yml:57]: can we add back in the comment that this package (and selenium) are optional dependencies of geoviews (just so we don't accidentally delete in the future). Can you also add pillow (another optional dependency of geoviews)

File: base_environment.yml [hunk 12]
```diff
@@ -1,110 +1,98 @@
...
+- pip=19.3
+- plotly=4.8
+- polyline=1.4
```
Comment (bolliger32) [inline base_environment.yml:60]: drop

File: base_environment.yml [hunk 13]
```diff
@@ -1,110 +1,98 @@
...
+- pygeos=0.6
```
Comment (bolliger32) [inline base_environment.yml:61]: drop this - it's a low-level library that other packages depend on

File: base_environment.yml [hunk 14]
```diff
@@ -1,110 +1,98 @@
...
+- pyinterp=0.3
+- pyshp=2.1
+- python=3.7
+- python-blosc=1.9
+- python-snappy=0.5
```
Comment (bolliger32) [inline base_environment.yml:66]: can drop - both included in numcodecs

File: base_environment.yml [hunk 15]
```diff
@@ -1,110 +1,98 @@
...
+- pyviz_comms=0.7
```
Comment (bolliger32) [inline base_environment.yml:67]: drop - installed with geoviews

File: base_environment.yml [hunk 16]
```diff
@@ -1,110 +1,98 @@
...
+- scikit-image=0.17
+- scikit-learn=0.23
+- scipy=1.4
+- seaborn=0.10
+- selenium=3.141
+- shapely=1.7
+- sparse=0.9
+- statsmodels=0.11
+- tini=0.18
+- unzip=6.0
```
Comment (bolliger32) [inline base_environment.yml:81]: drop b/c now installed earlier

File: base_environment.yml [hunk 17]
```diff
@@ -1,110 +1,98 @@
...
+- uritemplate=3.0
```
Comment (bolliger32) [inline base_environment.yml:82]: drop

File: base_environment.yml [hunk 18]
```diff
@@ -1,110 +1,98 @@
...
+- xarray=0.15
+- xesmf=0.3
+- xgcm=0.3
```
Comment (bolliger32) [inline base_environment.yml:85]: drop

File: base_environment.yml [hunk 19]
```diff
@@ -1,110 +1,98 @@
...
+- xhistogram=0.1
+- xlrd=1.2
+- xrft=0.2
```
Comment (bolliger32) [inline base_environment.yml:88]: drop

File: base_environment.yml [hunk 20]
```diff
@@ -1,110 +1,98 @@
...
+- zarr=2.4
+- zeromq=4.3
+- zict=2.0
```
Comment (bolliger32) [inline base_environment.yml:91]: drop - low-level dependencies that we probably dont need to pin

File: base_environment.yml (hunk 5 above)
Comment (dpa9694) [reply base_environment.yml]: would we rather have `esmpy==mpi` than the minor version? (`esmpy==7.1`)

Comment (bolliger32) [reply base_environment.yml]: nope I think we can do the minor version now! see [this page](https://xesmf.readthedocs.io/en/latest/installation.html?highlight=mpi#install-on-local-machine-with-conda)

**Comment by bolliger32** [comment]
🍅 🍷

File: base_environment.yml [hunk 21]
```diff
@@ -1,110 +1,81 @@
...
+- statsmodels=0.11.1=py37h8f50634_2
+- tini=0.18.0=h14c3975_1001
+- xarray=0.15.1=py_0
+- xesmf=0.3.0=py_0
+- xhistogram=0.1.1=py_0
+- xlrd=1.2.0=py_0
+- zarr=2.4.0=py_0
+- zeromq=4.3.2=he1b5a44_2
+- pip:
+  - py-noaa
```
Comment (bolliger32) [inline base_environment.yml:76]: add pins for pip-installed packages

**Comment by bolliger32** [comment]
wahoo! @delgadom @alihamidi1 assuming this PR merges, feel free to give the dev-pyTC-base:latest a try if you want to test out a new coastal notebook/worker pairing that includes VDATUM (and other package updates).

**Pull request merged**